.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,674 @@
+          mkdir -p ./packages/react-native/sdks/hermesc ./packages/react-native/sdks/hermesc/osx-bin ./packages/react-native/sdks/hermesc/win64-bin ./packages/react-native/sdks/hermesc/linux64-bin
+
+          # When build_hermes_macos runs as a matrix, it outputs
+          if [[ -d $HERMES_WS_DIR/osx-bin/Release ]]; then
+            cp -r $HERMES_WS_DIR/osx-bin/Release/* ./packages/react-native/sdks/hermesc/osx-bin/.
+          elif [[ -d $HERMES_WS_DIR/osx-bin/Debug ]]; then
+            cp -r $HERMES_WS_DIR/osx-bin/Debug/* ./packages/react-native/sdks/hermesc/osx-bin/.
+          else
+            ls $HERMES_WS_DIR/osx-bin || echo "hermesc macOS artifacts directory missing."
+            echo "Could not locate macOS hermesc binary."; exit 1;
+          fi
+
+          cp -r $HERMES_WS_DIR/win64-bin/* ./packages/react-native/sdks/hermesc/win64-bin/.
+          cp -r $HERMES_WS_DIR/linux64-bin/* ./packages/react-native/sdks/hermesc/linux64-bin/.
+          mkdir -p ./packages/react-native/ReactAndroid/external-artifacts/artifacts/
+          cp $HERMES_WS_DIR/hermes-runtime-darwin/hermes-ios-Debug.tar.gz ./packages/react-native/ReactAndroid/external-artifacts/artifacts/hermes-ios-debug.tar.gz
+          cp $HERMES_WS_DIR/hermes-runtime-darwin/hermes-ios-Release.tar.gz ./packages/react-native/ReactAndroid/external-artifacts/artifacts/hermes-ios-release.tar.gz
+          cp $HERMES_WS_DIR/dSYM/Debug/hermes.framework.dSYM  ./packages/react-native/ReactAndroid/external-artifacts/artifacts/hermes-framework-dSYM-debug.tar.gz
+          cp $HERMES_WS_DIR/dSYM/Release/hermes.framework.dSYM  ./packages/react-native/ReactAndroid/external-artifacts/artifacts/hermes-framework-dSYM-release.tar.gz
+      - name: Use Node.js 18
+        uses: actions/setup-node@v4.0.0
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --non-interactive
+      - name: Build packages
+        run: yarn build
+      # Continue with publish steps
+      - name: Set npm credentials
+        if: needs.set_release_type.outputs.RELEASE_TYPE == 'release' ||
+            needs.set_release_type.outputs.RELEASE_TYPE == 'nightly'
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.CIRCLE_NPM_TOKEN }}" > ~/.npmrc
+      - name: Publish NPM
+        shell: bash
+        run: |
+          echo "GRADLE_OPTS = $GRADLE_OPTS"
+          # We can't have a separate step because each command is executed in a separate shell
+          # so variables exported in a command are not visible in another.
+          if [[ "${{ needs.set_release_type.outputs.RELEASE_TYPE }}" == "dry-run" ]]; then
+            export ORG_GRADLE_PROJECT_reactNativeArchitectures="arm64-v8a"
+          else
+            export ORG_GRADLE_PROJECT_reactNativeArchitectures="armeabi-v7a,arm64-v8a,x86,x86_64"
+          fi
+          node ./scripts/releases-ci/publish-npm.js -t ${{ needs.set_release_type.outputs.RELEASE_TYPE }}
```

Comment:
This jobs publish stuff to Maven. I found that we read credentials [here](https://github.com/facebook/react-native/blob/main/build.gradle.kts#L49-L50), but from where are they read?
On CCI, I found these 4 variables:

* `ORG_GRADLE_PROJECT_SIGNING_KEY_ENCODED`
* `ORG_GRADLE_PROJECT_SIGNING_PWD`
* `ORG_GRADLE_PROJECT_SONATYPE_PASSWORD`
* `ORG_GRADLE_PROJECT_SONATYPE_USERNAME`

But I can't read their values, of course. I think we have to regenerate them, right? @cortinico